pages/common/wget.md

```diff
@@ -27,14 +27,10 @@
 
 `wget --ftp-user={{username}} --ftp-password={{password}} {{ftp://example.com}}`
 
-- Limit download speed to 200 kB/s:
-
-`wget --limit-rate={{200k}} {{https://example.com}}`
-
 - Continue an incomplete download:
 
 `wget -c {{https://example.com}}`
 
-- Retry a given number of times if the download doesn't succeed at first:
+- Make the progress bar disappear:
```

Comment:
We usually prefer a formal tone, and it would be good to start a word with the flag. Perhaps - `Enable quiet mode to suppress output` is better. This also clarifies that not only the progress bar is hidden but all error messages and information.